.github/workflows/ci.yml

```
@@ -18,10 +18,15 @@ jobs:
         # Fail build if there are warnings
         # build with TLS just for compilation coverage
         run: make -j4 all-with-unit-tests SERVER_CFLAGS='-Werror' BUILD_TLS=yes USE_FAST_FLOAT=yes
+      - name: install old server for compatibility testing
```

Since I guess we might theoretically do other types of compatibility, maybe something like:
```suggestion
      - name: install secondary server for compatibility testing
```

---

I changed to "other" in the test framework. In this particular case, the CI job is testing against 7.2.7 which is actually an old version, so I kept "old" here.

We can change it to "other" or "secondary" in the future, if we change the CI job to use some matrix of versions or something like that. Sounds good?

---

No strong opinion from me, the current wording makes sense.